### ISSUE TITLE
BCO OpenHAB binding always authenticated as admin

### ISSUE THREAD

**Issue by DivineThreepwood** [openbase/bco.dal#146]
Whatever I try, the binding seems no to accept the credentials.
@pLeminoq Could you confirm this behaviour in you setup as well?

To finalize the authentication module, I would like to avoid the admin, admin login.

**Comment by pLeminoq** [comment]
Yes, I can reproduce this. The underlying issue is that the binding uses the admin credentials as a backup. Most likely the credentials configured for the binding do not match or are not valid Base64 encoded. I do not know why but the logging from the binding informing about this will only appear in the OpenHAB console and not in the log file. Therefore, you do not receive any error messages if you start it as a service.

Would you rather have that the binding does not login at all if the credentials are not valid? Additionally, I think we need a better way of configuring the openhab user credentials so that the issue with the Base64 encoding does not happen...

**Comment by DivineThreepwood** [comment]
However, accessing the credentials manually via the credential store is not a nice solution anyway.
* [ ] The bco console should admins allow to print the encrypted credentials of a certain user which can than be used as api token.
* [ ] In long term bcozy should offer a way to generate api tokens. I created a dedicated issue for that. openbase/bco.bcozy#66
* [ ] We need a part in the bco doc "how to setup openhab" and there we need to describe step by step how to setup the credentials.
* [ ] The default admin admin authentication for the binding should be fine as long as:
  * [ ] a warning is printed during openhab startup or alternative displaying it on the paper ui (if supported) that the credential setup is still missing.
  * [ ] The default admin password has not been changed yet, otherwise the admin login should be avoided.
